sopel/bot.py

```diff
@@ -597,6 +597,20 @@ def _nick_blocked(self, nick):
         return False
 
     def _shutdown(self):
+        self.scheduler.stop()
+
+        stderr(
+            'Stopping the Job Scheduler.'
```

Comment:
Would prefer to print the message, then run the action, instead of the current "backwards" ordering.